media/harvester/test/20210623093518719032.watch.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html><html lang="de" dir="ltr"><head><style nonce="60uX9wwTntq6TkotpSa5OQ">
```

Comment:
Zijn al deze files nodig voor tests?

Reply:
Ja, ik heb nu voor alle materialen die we gebruiken in de tests de bronbestanden toegevoegd. De paths van de resources wijzen nu ook daadwerkelijk naar deze files i.p.v. een niet bestaande file (dat geeft wat gekke errors links en rechts)